graph/src/components/store.rs

```diff
@@ -829,7 +829,11 @@ impl<DS: DataSource> From<&DS> for StoredDynamicDataSource {
         Self {
             name: ds.name().to_owned(),
             source: ds.source().clone(),
-            context: ds.context().map(|ctx| serde_json::to_string(&ctx).unwrap()),
+            context: ds
```

Comment:
Ultimately, the data model for dds is still pretty eth dependent; we might have to change it to something that just stores the dds as a JSONB blob to get the flexibility we need; that should be fine since we never query for individual dds attributes.

Reply:
Yes there are decisions to be made here, storing as jsonb is an interesting idea.